scripts/cbmc_json_parser.py

```diff
@@ -270,6 +275,9 @@ def postprocess_results(properties):
     annotate_properties_with_reach_results(properties, reach_checks)
     remove_check_ids_from_description(properties)
 
+    if not extra_ptr_check:
+        properties = filter_ptr_checks(properties)
```

Comment:
Can you clarify why this filtering is needed if we're turning off both `--pointer-overflow-check` and `--pointer-primitive-check`?

Reply:
Some checks that have the pointer arithmetic class id (the relation ones) are actually controlled by the --pointer-check option. :(

I didn't quite needed to check for PTR primitives, but I figured I should keep it for completeness.

Reply:
That's a bummer. We should probably file a CBMC issue to allow more fine-grained control of pointer checks.